test/Flick/AnimatedCarousalPage.xaml.cs

```diff
@@ -71,42 +149,91 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
 
         private void Repeater_Loaded(object sender, RoutedEventArgs e)
         {
-            sv.ChangeView((layout.ItemWidth + layout.Spacing) * 500, null, null, true);
+            sv.ChangeView(((layout.ItemWidth + layout.Spacing) * 500), null, null, true);
```

Comment:
>500 [](start = 65, length = 3)

would be good to take this out as a constant that is defined in this class.

Reply:
I agree. We need to expose a property on the Layout that a client can use to determine the Item count used by the repeater.

Reply:
That can be done in a later iteration.